### PR TITLE
Benchmark improvements

### PR DESCRIPTION
**Pull request by mbertuletti**
Add a python script that parses the csv files in one of the folders in results, to generate average performances over all the cores
.

### CODE REVIEW

File: hardware/scripts/gen_avg.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+         'itl_stores_global']
+
+for files in os.listdir(path):
+    if files.endswith(ext):
+        print(files)
+        csvread = pd.read_csv(files)
+        print('\n\n')
+        #print(csvread)
+        print("******************************")
+        print("**    AVERAGE PERFORMANCES  **")
```
Comment (suehtamacv) [inline hardware/scripts/gen_avg.py:46]: Would `Performance` work better?

File: hardware/scripts/gen_avg.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import csv
+import os
+import pandas as pd
+import numpy as np
+from collections import deque, defaultdict
+import sys
+
+path = '/scratch2/mempool/hardware/build/traces'
```
Comment (suehtamacv) [inline hardware/scripts/gen_avg.py:8]: Do we need to have this hardcoded path in the script? How do you call this python script? Maybe the `path` could be a parameter?

**Review by suehtamacv — CHANGES_REQUESTED** [review]
The linter has some issues with your Python code, and there is a parameter in the python script which probably should not be hardcoded, but in general it looks good!

File: hardware/scripts/gen_avg.py (hunk 2 above)
Comment (mbertuletti) [reply hardware/scripts/gen_avg.py:8]: No, we don't need to hardcode this, thanks

**Pull request closed without merging**